### PR TITLE
Version 1.6.39 

### PR DESCRIPTION
**Pull request by akshayurankar48**
### Description
Updated PHP version number in the readme file. 

### Checklist:
- [x] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,6 +140,9 @@ This same applies when you are creating your Header/Footer using this plugin.
 
 ## Changelog ##
 
+### 1.6.38.1 ###
+- Improvement: Updated PHP version number in the README file.
```
Comment (sushmak02) [inline README.md:144]: Updated minimum PHP version requirement to 7.4.

File: readme.txt [hunk 2]
```diff
@@ -140,6 +140,9 @@ This same applies when you are creating your Header/Footer using this plugin.
 
 == Changelog ==
 
+= 1.6.38.1 =
```
Comment (sushmak02) [inline readme.txt:143]: 1.6.39

File: README.md (hunk 1 above)
Comment (akshayurankar48) [reply README.md]: updated @sushmak02

File: readme.txt (hunk 2 above)
Comment (akshayurankar48) [reply readme.txt]: done @sushmak02

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 **Tags:** elementor, header footer builder, header template, footer template, elementor widget  
 **Donate link:** https://www.paypal.me/BrainstormForce  
 **Requires at least:** 4.4  
-**Requires PHP:** 5.4  
+**Requires PHP:** 7.4  
 **Tested up to:** 6.6  
 **Stable tag:** 1.6.38
```
Comment (vijetaR) [inline README.md:8]: Update this

Comment (akshayurankar48) [reply README.md]: updated @vijetaR

**Review by vijetaR — APPROVED** [review]

**Review by sushmak02 — APPROVED** [review]

**Pull request merged**